app/migrations/Version20231120143030.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20231120143030 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
```

Comment:
I like having a description so `console doctrine:migrations:list` can help to find a migration:

```suggestion
        return 'Add ldap_alias_field column to the connector table.';
```